prog-o-meter.py

```diff
@@ -352,7 +392,175 @@ def save_and_close(self, event=None):
     def get_name(self):
         """Return the username. """
         return self.username
+    
+class PersonalizationGUI(object):
+
+    """Class contains all about the GUI of the personalization Window.
+
+    Attributes:
+        root: a tkinter root.
+        user: the user using the program
+        logname: the user logname used in the program
+    """
+
+    def __init__(self, user, logname):
+        """Opens a Tkinter window for personalization.
+
+        The window contains all the buttons and entries
+        needed for personalization the prog-o-meter program
+        It contains 3 tabs for each of the personalizable items
+        and 3 buttons for getting it back to "Default",
+        "Apply" the values and getting "Back" to the main window
+        """
+        # Attributes
+        self.user = user
+        self.logname = logname
+        self.root = Tk.Tk()
+        self.tabs_layout()
+        self.colors_layout()
+        self.motivation_layout()
+        self.days_layout()
+        self.root.mainloop()
+
+
+    def tabs_layout(self):
+        """Creates the Notebook widget, all tabs are children of the root_tab
+
+        Creates 3 tabs:
+            colors_tab, motivation_tab and days_tab
+            Each of them has a function for organization
+        Create 3 Buttons:
+            defaults_button: 
+                when pressed uses the default_values function
+            apply_button:    
+                when pressed uses the apply_values function
+            back_button:     
+                when pressed uses the go_back function
+        Obs:
+            The Notebook widget cant be used without ttk,
+            Every other thing has it Tk counterpart
+        """
+        root_tab = ttk.Notebook(self.root)
+        self.colors_tab = ttk.Frame(root_tab)   
+        self.motivation_tab = ttk.Frame(root_tab)   
+        self.days_tab = ttk.Frame(root_tab)
+        root_tab.add(self.colors_tab, text="Colors")
+        root_tab.add(self.motivation_tab, text="Motivations")
+        root_tab.add(self.days_tab, text="Days")
+        root_tab.grid(row=0, column=0, columnspan=3)
+        defaults_button = ttk.Button(self.root, text="Default",
+                                     command=self.default_values)
+        defaults_button.grid(row=1, column=0, sticky="WE")
+        apply_button = ttk.Button(self.root,
+                                  text="Apply",command=self.apply_values)
+        apply_button.grid(row=1, column=1, sticky="WE")

```

Comment:
I haven't worked with ttk, so I am excited to learn more about it. Would you mind explaining what the sticky keyword does? :)

Reply:
cool, thanks :)